### PR TITLE
Plural of goose = geese

### PR DESCRIPTION
**Pull request by renedekat**

### CODE REVIEW

File: lib/Doctrine/Common/Inflector/Inflector.php [hunk 1]
```diff
@@ -87,9 +87,10 @@ class Inflector
             'echo' => 'echoes',
             'foot' => 'feet',
             'fungus' => 'fungi',
-            'ganglion' => 'ganglions',
+            'ganglion' => 'ganglions',
```
Comment (stof) [inline lib/Doctrine/Common/Inflector/Inflector.php]: should be reverted

Comment (renedekat) [inline lib/Doctrine/Common/Inflector/Inflector.php]: Done

**Comment by Ocramius** [comment]
Needs additions to the test suite too

**Comment by HipsterJazzbo** [comment]
Fixed with tests in #67

**Comment by malarzm** [comment]
Closing in favour of #67

**Pull request closed without merging**